File: assignment 10 - C# practice problems/14.cs
//Write a c# program to concatenate two strings except their first character.
using System;
namespace MyNameSpace
{
    class Program14
    {
        public static void StringConcatenateDriverFunction()
        {

            Console.WriteLine("Enter two strings: ");
            string stringInput1 = Console.ReadLine();
            string stringInput2 = Console.ReadLine();

            stringInput1 = stringInput1.Substring(1, stringInput1.Length - 1);
            stringInput2 = stringInput2.Substring(1, stringInput2.Length - 1);

            Console.WriteLine(stringInput1 + stringInput2);


        }
    }
}

File: assignment 10 - C# practice problems/js/14.js
//Write a JavaScript program to concatenate two strings except their first character.
function stringConcatenateDriverFunction() {
    let stringInput1 = prompt("Enter a string1");
    let stringInput2 = prompt("Enter a string2");

    if (stringInput1 != null && stringInput2 != null) {
        stringInput1 = stringInput1.trim();
        stringInput2 = stringInput2.trim();

        stringInput1 = stringInput1.substr(1, stringInput1.length);
        stringInput2 = stringInput2.substr(1, stringInput2.length);

        alert(stringInput1.concat(stringInput2));
    }
}

stringConcatenateDriverFunction()